samples/Cryptocash.Api/Program.cs

```diff
@@ -1,33 +1,24 @@
 using Nox;
-using Serilog;
 
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
 builder.Services.AddControllers();
 
-// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
-builder.Services.AddSwaggerGen();
-
-builder.AddNox();
 
+builder.AddNox(opts => opts.WithSwagger());
```

Comment:
IMO this should be the default behavior of AddNox, we should allow to disable swagger instead